Undeclared BLOSC_STUNE

**Describe the bug**
When I try to build PyTables with python-blosc v2.2.2 I have a build error saying the
```
[cut].venv/include/blosc2.h:1109:21: error: ‘BLOSC_STUNE’ undeclared here (not in a function); did you mean ‘BLOSC_LAST_TUNE’?
 1109 | static int g_tune = BLOSC_STUNE;
      |                     ^~~~~~~~~~~
      |                     BLOSC_LAST_TUNE
```

The issue seems to be related to the fact that the enumeration `BLOSC_STUNE` is defined only if `BLOSC_H` is defined

https://github.com/Blosc/c-blosc2/blob/4e78fbb9dccf7563cd93d36821c696d06b302972/include/blosc2.h#L213-L221

while it is used without any check on `BLOSC_H`

https://github.com/Blosc/c-blosc2/blob/4e78fbb9dccf7563cd93d36821c696d06b302972/include/blosc2.h#L1109

**To Reproduce**
```
$ git clone https://github.com/PyTables/PyTables.git
$ cd PyTables
$ python3 -m venv .venv --prompt venv
$ source .venv/bin/activate
$ python3 -m pip install numpy cython numexpr blosc blosc2 packaging py-cpuinfo
$ python3 setup.py build_ext --inplace
```

**Expected behavior**
PyTables builds correctly

**Logs**
```
$ python3 setup.py build_ext --inplace
* Using Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
* Found cython 0.29.34
* USE_PKGCONFIG: True
* pkg-config header dirs for HDF5: /usr/include/hdf5/serial
* pkg-config library dirs for HDF5: /usr/lib/x86_64-linux-gnu/hdf5/serial
* Found HDF5 headers at ``/usr/include/hdf5/serial``, library at ``/usr/lib/x86_64-linux-gnu/hdf5/serial``.
.. WARNING:: Could not find the HDF5 runtime.
   The HDF5 shared library was *not* found in the default library
   paths. In case of runtime problems, please remember to install it.
* Found LZO 2 headers at ``/usr/include``, the library is located in the standard system search dirs.
* Skipping detection of LZO 1 since LZO 2 has already been found.
* Found bzip2 headers at ``/usr/include``, the library is located in the standard system search dirs.
* Found blosc headers at ``/usr/include``, the library is located in the standard system search dirs.
* Found blosc2 headers at ``/home/antonio/projects/PyTables/.venv/include``, library at ``/home/antonio/projects/PyTables/.venv/lib``.
  * Copying blosc2 runtime library to 'tables' dir because it was not found in standard locations
Compiling tables/utilsextension.pyx because it changed.
[1/1] Cythonizing tables/utilsextension.pyx
Compiling tables/hdf5extension.pyx because it changed.
[1/1] Cythonizing tables/hdf5extension.pyx
Compiling tables/tableextension.pyx because it changed.
[1/1] Cythonizing tables/tableextension.pyx
Compiling tables/linkextension.pyx because it changed.
[1/1] Cythonizing tables/linkextension.pyx
Compiling tables/_comp_lzo.pyx because it changed.
[1/1] Cythonizing tables/_comp_lzo.pyx
Compiling tables/_comp_bzip2.pyx because it changed.
[1/1] Cythonizing tables/_comp_bzip2.pyx
Compiling tables/lrucacheextension.pyx because it changed.
[1/1] Cythonizing tables/lrucacheextension.pyx
Compiling tables/indexesextension.pyx because it changed.
[1/1] Cythonizing tables/indexesextension.pyx
/home/antonio/projects/PyTables/.venv/lib/python3.11/site-packages/setuptools/config/pyprojecttoml.py:66: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
  config = read_configuration(filepath, True, ignore_option_errors, dist)
running build_ext
building 'tables.utilsextension' extension
creating build
creating build/temp.linux-x86_64-cpython-311
creating build/temp.linux-x86_64-cpython-311/hdf5-blosc
creating build/temp.linux-x86_64-cpython-311/hdf5-blosc/src
creating build/temp.linux-x86_64-cpython-311/hdf5-blosc2
creating build/temp.linux-x86_64-cpython-311/hdf5-blosc2/src
creating build/temp.linux-x86_64-cpython-311/src
creating build/temp.linux-x86_64-cpython-311/tables
x86_64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DNDEBUG=1 -DHAVE_LZO2_LIB=1 -DHAVE_BZ2_LIB=1 -DHAVE_BLOSC_LIB=1 -DHAVE_BLOSC2_LIB=1 -Ihdf5-blosc2/src -I/home/antonio/projects/PyTables/.venv/include -Ihdf5-blosc/src -I/usr/include/hdf5/serial -I/home/antonio/projects/PyTables/.venv/lib/python3.11/site-packages/numpy/core/include -I/home/antonio/projects/PyTables/.venv/include -I/usr/include/python3.11 -c hdf5-blosc/src/blosc_filter.c -o build/temp.linux-x86_64-cpython-311/hdf5-blosc/src/blosc_filter.o -Isrc -DH5_USE_18_API -DH5Acreate_vers=2 -DH5Aiterate_vers=2 -DH5Dcreate_vers=2 -DH5Dopen_vers=2 -DH5Eclear_vers=2 -DH5Eprint_vers=2 -DH5Epush_vers=2 -DH5Eset_auto_vers=2 -DH5Eget_auto_vers=2 -DH5Ewalk_vers=2 -DH5E_auto_t_vers=2 -DH5Gcreate_vers=2 -DH5Gopen_vers=2 -DH5Pget_filter_vers=2 -DH5Pget_filter_by_id_vers=2 -DH5Tarray_create_vers=2 -DH5Tget_array_dims_vers=2 -DH5Z_class_t_vers=2 -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
x86_64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DNDEBUG=1 -DHAVE_LZO2_LIB=1 -DHAVE_BZ2_LIB=1 -DHAVE_BLOSC_LIB=1 -DHAVE_BLOSC2_LIB=1 -Ihdf5-blosc2/src -I/home/antonio/projects/PyTables/.venv/include -Ihdf5-blosc/src -I/usr/include/hdf5/serial -I/home/antonio/projects/PyTables/.venv/lib/python3.11/site-packages/numpy/core/include -I/home/antonio/projects/PyTables/.venv/include -I/usr/include/python3.11 -c hdf5-blosc2/src/blosc2_filter.c -o build/temp.linux-x86_64-cpython-311/hdf5-blosc2/src/blosc2_filter.o -Isrc -DH5_USE_18_API -DH5Acreate_vers=2 -DH5Aiterate_vers=2 -DH5Dcreate_vers=2 -DH5Dopen_vers=2 -DH5Eclear_vers=2 -DH5Eprint_vers=2 -DH5Epush_vers=2 -DH5Eset_auto_vers=2 -DH5Eget_auto_vers=2 -DH5Ewalk_vers=2 -DH5E_auto_t_vers=2 -DH5Gcreate_vers=2 -DH5Gopen_vers=2 -DH5Pget_filter_vers=2 -DH5Pget_filter_by_id_vers=2 -DH5Tarray_create_vers=2 -DH5Tget_array_dims_vers=2 -DH5Z_class_t_vers=2 -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
In file included from hdf5-blosc2/src/blosc2_filter.h:8,
                 from hdf5-blosc2/src/blosc2_filter.c:16:
/home/antonio/projects/PyTables/.venv/include/blosc2.h:1109:12: warning: ‘g_tune’ defined but not used [-Wunused-variable]
 1109 | static int g_tune = BLOSC_STUNE;
      |            ^~~~~~
/home/antonio/projects/PyTables/.venv/include/blosc2.h:1056:21: warning: ‘BLOSC2_IO_CB_DEFAULTS’ defined but not used [-Wunused-variable]
 1056 | static blosc2_io_cb BLOSC2_IO_CB_DEFAULTS = {0};
      |                     ^~~~~~~~~~~~~~~~~~~~~
x86_64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DNDEBUG=1 -DHAVE_LZO2_LIB=1 -DHAVE_BZ2_LIB=1 -DHAVE_BLOSC_LIB=1 -DHAVE_BLOSC2_LIB=1 -Ihdf5-blosc2/src -I/home/antonio/projects/PyTables/.venv/include -Ihdf5-blosc/src -I/usr/include/hdf5/serial -I/home/antonio/projects/PyTables/.venv/lib/python3.11/site-packages/numpy/core/include -I/home/antonio/projects/PyTables/.venv/include -I/usr/include/python3.11 -c src/H5ARRAY.c -o build/temp.linux-x86_64-cpython-311/src/H5ARRAY.o -Isrc -DH5_USE_18_API -DH5Acreate_vers=2 -DH5Aiterate_vers=2 -DH5Dcreate_vers=2 -DH5Dopen_vers=2 -DH5Eclear_vers=2 -DH5Eprint_vers=2 -DH5Epush_vers=2 -DH5Eset_auto_vers=2 -DH5Eget_auto_vers=2 -DH5Ewalk_vers=2 -DH5E_auto_t_vers=2 -DH5Gcreate_vers=2 -DH5Gopen_vers=2 -DH5Pget_filter_vers=2 -DH5Pget_filter_by_id_vers=2 -DH5Tarray_create_vers=2 -DH5Tget_array_dims_vers=2 -DH5Z_class_t_vers=2 -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
In file included from hdf5-blosc2/src/blosc2_filter.h:8,
                 from src/H5ARRAY.c:8:
/home/antonio/projects/PyTables/.venv/include/blosc2.h:1109:21: error: ‘BLOSC_STUNE’ undeclared here (not in a function); did you mean ‘BLOSC_LAST_TUNE’?
 1109 | static int g_tune = BLOSC_STUNE;
      |                     ^~~~~~~~~~~
      |                     BLOSC_LAST_TUNE
/home/antonio/projects/PyTables/.venv/include/blosc2.h:1109:12: warning: ‘g_tune’ defined but not used [-Wunused-variable]
 1109 | static int g_tune = BLOSC_STUNE;
      |            ^~~~~~
/home/antonio/projects/PyTables/.venv/include/blosc2.h:1056:21: warning: ‘BLOSC2_IO_CB_DEFAULTS’ defined but not used [-Wunused-variable]
 1056 | static blosc2_io_cb BLOSC2_IO_CB_DEFAULTS = {0};
      |                     ^~~~~~~~~~~~~~~~~~~~~
error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
```

**System information:**
 - OS: GNU/Linux Ubuntu 23.04
 - Compiler gcc 12.2
 - Version blosc2 >= 2.9.0

--- a/include/blosc2.h
+++ b/include/blosc2.h
@@ -211,9 +211,7 @@ enum {
  * @brief Codes for the different tunes shipped with Blosc
  */
 enum {
-#ifndef BLOSC_H
     BLOSC_STUNE = 0,
-#endif // BLOSC_H
     BLOSC_LAST_TUNE = 1,
     //!< Determine the last tune defined by Blosc.
     BLOSC_LAST_REGISTERED_TUNE = BLOSC2_GLOBAL_REGISTERED_TUNE_START + BLOSC2_GLOBAL_REGISTERED_TUNES - 1,

